setup/lib/utils.js

```diff
@@ -0,0 +1,74 @@
+const { setFailed } = require('@actions/core');
+const os = require('os');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const semver = require('semver');
+const core = require('@actions/core');
+
+// arch in [arm, x32, x64...] (https://nodejs.org/api/os.html#os_os_arch)
+// return value in [amd64, 386, arm]
+function mapArch(arch) {
+    const mappings = {
+        x32: '386',
```

Comment:
Looks like Notation doesn't have `x32` binary according to the [release page](https://github.com/notaryproject/notation/releases). We only have `amd64` and `arm64`

Reply:
Add an arch const with the value of amd64 and removed the mapArch function. :)